src/utils/decoder_util.h

```diff
@@ -561,24 +561,23 @@ class DecoderUtil {
     // need to do for res.
     static void softmaxTile(float *AB, float *sum, float *max, float *preSum, float *preMax, float refac,
             const float *attnMask, int m, int k, int attnMskStride) {
-        float max_val = std::numeric_limits<float>::lowest();
+        float minVal = std::numeric_limits<float>::lowest();
```

Comment:
I think the naming of maxVal is better than minVal, as it represents the maximum elements in that channel.

Reply:
Done~